win32/test/win32crypt_testcert_ctlcontext.cer

```diff
@@ -0,0 +1 @@
+MIIDKjCCAhKgAwIBAgIQaywn/6CWRlGT/OAEKRt05TANBgkqhkiG9w0BAQsFADASMRAwDgYDVQQDEwdicmFuZG9uMB4XDTIyMDQxMjE1MjQyOFoXDTIzMDQxMjE1MzQyOFowEjEQMA4GA1UEAxMHYnJhbmRvbjCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAMTXRWxv/xyYMy7BIUj6obtQ9+prBV+6dgmvIXMV5OqirGb2ldA3lP2yO1VSnxchFSLuwGNsEc9R6MYwa+ftPuJMQETzCpQTDZqmu8LqZAxkqU6D4sqHWU3DhSTE0M1vBBTFN8vSvrhAAnH0EJ1sEfIHsxOMzksEMWxFVzR6yMwgTZe1akPq5AfthSdKmN+XI3nIliiJbeq1SpFvNrj5tpdp3xqr+G4wjuBzghgmMAE4C/xAQ9b43VAbQvQiw5JSFU9sAQeUvFdfDLg/tVN5aDmJUdVLQ9oXMAJv1ZfJGdpPwEsGGhnHx0Qrvpn1jh56Nq7tDhsalgkoJw2HhPfVwvUCAwEAAaN8MHowDgYDVR0PAQH/BAQDAgWgMAkGA1UdEwQCMAAwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMCMB8GA1UdIwQYMBaAFPHRgQjh+QpSvNiXHdqWbSwIquZ7MB0GA1UdDgQWBBTx0YEI4fkKUrzYlx3alm0sCKrmezANBgkqhkiG9w0BAQsFAAOCAQEAIvLp6BqDiI9Hh40b1+EOIJetX0pf6gyi16N/s7d+fR3hHxerfLxJ4tB2FR1YCx+xf1h66ucaPEBHaokOyQ+UOuLcoc0KHOv+MIzZ5G8d7lv7AGpPn3F0OwwRuo4SxJNgxRYYynOfQf5QTs5rJkGBmMRehM6tSIlZWymFvsBxxCjI+SIiRa7/iL1TEUDMbMQI6MG0i8uLrOHGO7ioChk+rgXEH82gl6nPQx8pQrVTzv2NAJXVPWNhrGci/64b0OqJ4WFUrqbPXH7Vp27ROUD4PfB9CgVxBA+wjDwHeOAMmKxUvA1jGwyqf0VE11mJaSxH2luNv7/z+dGSUQDEXCqWlw==
```

Comment:
```
...
DBTE>>> cert['ContentType']  
1      # CERT_QUERY_CONTENT_CERT  The content is a single certificate. 
DBTE>>> cs1=cert['CertStore']
DBTE>>> cs1.CertEnumCTLsInStore()
[]
DBTE>>> cs1.CertEnumCertificatesInStore()
[<PyCERT_CONTEXT object at 0x0000021613956B90>]
```

=> This file seems not to have a CTL context but only a single cert in base64 format